src/ArduinoIoTCloudTCP.cpp

```diff
@@ -763,14 +763,16 @@ void ArduinoIoTCloudTCP::sendPropertiesToCloud()
 }
 
 #if OTA_ENABLED
-void ArduinoIoTCloudTCP::sendOTAPropertiesToCloud()
+void ArduinoIoTCloudTCP::sendOTAPropertiesToCloud(bool include_ota_req)
```

Comment:
```diff
-bool include_ota_req
+bool const include_ota_req
```
Minor suggestions, basically just use [const-correctness](https://isocpp.org/wiki/faq/const-correctness) as much as possible.

Reply:
i'll stick a post-it on my monitor so i will not forget it anymore :sweat_smile:

Reply:
i'm lucky enough this change was outdated and not included in the final changes